src/domains/chart/utils/transformDataAttributes.ts

```diff
@@ -318,6 +320,7 @@ const getAttributesMap = (): AttributesMap => ({
   units: { key: "units" },
   unitsCommon: { key: "common-units" },
   unitsDesired: { key: "desired-units" },
+  aggr_method: { key: "aggr-method", defaultValue: "avg" },
```

Comment:
You should use the `mapDefaultAggrMethod` here.

Reply:
This is not possible to be used here

Reply:
OK, great, but we need to pass a default value based on the chart unit. Where do you think we should put this kind of logic?